lstchain/reco/reconstructor.py

```diff
@@ -299,7 +342,7 @@ def __call__(self, event, telescope_id, dl1_container):
 
             return self.predict(unit_cam, fit_params)
 
-    def clean_data(self, pix_x, pix_y, pix_radius, times, start_parameters, telescope_id):
+    def clean_data(self, image, geometry, pix_x, pix_y, pix_radius, times, start_parameters, telescope_id):
         """
         Method used to select pixels and time samples used in the fitting procedure.
```

Comment:
Can you update the docstring to reflect the additional support for DVR parameters?